exonum/src/api/node/public/explorer.rs

```diff
@@ -171,6 +171,23 @@ impl AsRef<[u8]> for TransactionHex {
     }
 }
 
+/// Status of a call.
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct CallStatusResponse {
+    /// Call status
+    pub status: ExecutionStatus,
+}
+
+/// Call status query parameters.
+/// To check `before_transactions` or `after_transactions` call
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct CallStatusQuery {
+    /// Height of a block
+    pub height: Height,
+    /// Numerical service identifier.
+    pub id: u32,
```

Comment:
Nit: Using `InstanceId` instead of `u32` would be more idiomatic. Same for `service_id` instead of `id`.